secure-boot-man.md

```diff
@@ -0,0 +1,121 @@
+SECURE-BOOT 8 "December 2020" "secure-boot 1.5.0" "User Manual"
+==================================================
+
+# NAME
+  secure-boot - UEFI SecureBoot on ArchLinux
+
+# SYNOPSIS
+  secure-boot [OPTIONS] \<command\>
+
+# DESCRIPTION
+`secure-boot` is a userspace application to use UEFI secure boot on ArchLinux.
+It generates keys to sign the bootimage, enrolls the keys into the UEFI
+firmware (experimental), signs the bootimage and creates EFI boot entries (no
+intermediate bootloader is needed).
+
+This process is fully configurable by putting the options into
+`/etc/secure-boot/config.mk` (in makefile format) or specify them on the
+command line (see `EXAMPLES`).
+
+Secure boot must be enabled in the UEFI firmware.
+
+# OPTIONS
+ESP=\<path\>
```

Comment:
should these be made code sections? in backticks?

seems to look better in the markdown rendering, and avoids the need for escaping the `<>`.

Reply:
I found this now: https://liw.fi/manpages/

> Fonts are again used to clarify things:
>
>   the name of the option (including any dashes at the beginning) is in bold
>   any argument is in italic, but the equals sign (if any) is in normal font
>   the comma between options is in normal font

I will fix this.

Reply:
It looks like everyone does it a bit different. `man ls` used bold options and italic upper case parameters, while `man git`  uses normal font for options and parameters.

Though I think it looks pretty good with bold options and italic parameters.